dns-rc-autoscaler/nanny/kubernetes_client.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nanny
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	api "k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_3"
+)
+
+const replicationControllerKind = "ReplicationController"
```

Comment:
Also https://github.com/kubernetes/kubernetes/pull/29551 will help a lot when it's complete.